### PR TITLE
Change the sample to use concat instead of startWith #27

### PR DESCRIPTION
**Pull request by alin-turcu**
Change the sample to use concat instead of startWith #27

### CODE REVIEW

File: grox-sample-rx2/src/main/java/com/groupon/grox/sample/RefreshColorCommand.java [hunk 1]
```diff
@@ -42,14 +42,20 @@
   //don't forget to convert errors in actions
   @Override
   public Observable<? extends Action> actions() {
-    return getColorFromServer()
-        .subscribeOn(io())
-        .map(ChangeColorAction::new)
-        .cast(Action.class)
-        .onErrorReturn(ErrorAction::new)
-        .startWith(fromCallable(RefreshAction::new));
+      final Observable<Action> refresh = Observable.just(new RefreshAction());
```
Comment (stephanenicolas) [inline grox-sample-rx2/src/main/java/com/groupon/grox/sample/RefreshColorCommand.java]: static import

File: grox-sample-rx/src/main/java/com/groupon/grox/sample/RefreshColorCommand.java [hunk 2]
```diff
@@ -39,15 +40,21 @@
...
+
   @Override
   public Observable<? extends Action> actions() {
-    return getColorFromServer()
-        .subscribeOn(io())
-        .map(ChangeColorAction::new)
-        .cast(Action.class)
-        .onErrorReturn(ErrorAction::new)
-        .startWith(fromCallable(RefreshAction::new));
+    final Observable<Action> refresh = Observable.just(new RefreshAction());
```
Comment (stephanenicolas) [inline grox-sample-rx/src/main/java/com/groupon/grox/sample/RefreshColorCommand.java]: static import

**Comment by stephanenicolas** [comment]
RTM + 2 minor comments. Thanks for fixing it. Please also update the CHANGELOG and fix the matching issue.

File: grox-sample-rx/src/main/java/com/groupon/grox/sample/RefreshColorCommand.java (hunk 2 above)
Comment (saguinav) [reply grox-sample-rx/src/main/java/com/groupon/grox/sample/RefreshColorCommand.java]: **Minor suggestion:** Change it to *inline* style.

```java
return Observable.just(new RefreshAction())
    .concatWith(refreshColor());
```

Comment (alin-turcu) [reply grox-sample-rx/src/main/java/com/groupon/grox/sample/RefreshColorCommand.java]: This is tricky. We need to type the observable in order for this to work. ``Observable<Action>.just`` does not work :)

**Pull request merged**